driver-kncasic.c

```diff
@@ -72,6 +72,7 @@ struct knc_core_state {
 		int slot;
 		uint32_t nonce;
 	} last_nonce;
+	uint32_t last_nonce_verified;
```

Comment:
If duplicate nonces are normal, it makes sense to detect and suppress them. If they're indicative of a problem or rare, it may be better to let the user find out by not filtering them.